### DISCUSSION TITLE
모바일에서 크롬 창 화면에 띄워놓아야 작동이 되나요?

### DISCUSSION THREAD

**Discussion by numnabad** [kimytsc/covid-rest-vaccine-macro#91] [Q&A]
다른 앱 하고 있거나 화면이 꺼져있으면 작동이 안 되는건가요?

**Comment by kimytsc** [comment]
모바일은 활성화된 화면만 동작하기에 다른 작업을 하면 동작하지 않을 수 있습니다.

그래서 세컨폰이나 분할화면, 팝업화면 등을 이용해서 동작시켜놓으시는걸 추천드립니다.
